testnet/stacks-node/src/config.rs

```diff
@@ -1107,6 +1112,12 @@ pub struct FeeEstimationConfig {
     pub fee_estimator: Option<FeeEstimatorName>,
     pub cost_metric: Option<CostMetricName>,
     pub log_error: bool,
+    /// If using FeeRateFuzzer, the amount of random noise, as a percentage of the base value (in
```

Comment:
Will it be problematic if different nodes use different fuzzer fractions?  I'm inclined to think "no, unless they're wildly different," but I'm curious to hear your thoughts.

Reply:
I'm kind of unclear on what would happen if we didn't do the random fuzzing now, actually.

If people submitted the same fees, would that clog the system or just mean that the miners had to choose who wins arbitrarily?

Because, right now we are going to decide who gets which random fuzz arbitrarily. So, both would be arbitrary.

Reply:
The goal of fuzzing isn't to break ties -- it's so that in periods of congestion, there's a set of higher fee rate transactions in the mempool, which miners will end up preferring, which will in turn cause fee estimates to go up naturally.

Reply:
> Will it be problematic if different nodes use different fuzzer fractions? I'm inclined to think "no, unless they're wildly different," but I'm curious to hear your thoughts.

No, even if they're wildly different. These fee estimates are only used by clients to determine how to price their transactions. If someone decides to run a wildly fuzzy node, and use it for their transactions, _their_ transactions might be priced entirely wrong, but it'd have little to no impact on other users. Setting the fuzz setting correctly for commonly queried nodes would be important though.

Reply:
Oh right. Yeah that makes sense.